lms/djangoapps/commerce/tests/test_views.py

```diff
@@ -17,6 +18,8 @@
 from commerce.tests import TEST_BASKET_ID, TEST_ORDER_NUMBER, TEST_PAYMENT_DATA, TEST_API_URL, TEST_API_SIGNING_KEY
 from commerce.tests.mocks import mock_basket_order, mock_create_basket
 from course_modes.models import CourseMode
+from embargo.test_utils import restrict_course
+from enrollment.tests.test_views import UrlMixin
```

Comment:
Tests in this app should not depend on code defined in arbitrary test modules in another app.  If there is a piece of shared code to be used in different tests, please define it in a shared / "neutral" utility module and have both apps' tests import from that.  Though, as I said in my previous comment, though it is generally good to reuse functionality where possible, the url assembly step might be trivial enough to simply define it in two places.